adafruit_sgp30.py

```diff
@@ -75,6 +75,40 @@ def __init__(self, i2c, address=_SGP30_DEFAULT_I2C_ADDR):
             raise RuntimeError('SGP30 Not detected')
         self.iaq_init()
 
+        self._co2eq = None # pylint: disable=invalid-name
+        self._tvoc = None # pylint: disable=invalid-name
+        self._baseline_tvoc = None # pylint: disable=invalid-name
+        self._baseline_co2eq = None # pylint: disable=invalid-name
+
+
+    @property
+    def tvoc(self): # pylint: disable=invalid-name
+        """Total Volatile Organic Compound in parts per billion."""
+        self._tvoc = self.iaq_measure()[1]
+        return self._tvoc
```

Comment:
Just `return self.iaq_measure()[1]` and similar below.

Reply:
Changed - thanks (will keep this in mind next time)!